src/main/java/uk/gov/justice/digital/delius/data/api/OffenderAssessments.java

```diff
@@ -11,6 +11,10 @@
 @NoArgsConstructor
 @AllArgsConstructor
 public class OffenderAssessments {
+
     @ApiModelProperty(value = "Risk of Serious Recidivism")
-    private int rsrScore;
+    private Double rsrScore;
+    @ApiModelProperty(value = "Offender Group Reconviction Scale")
+    private Integer OGRSScore;
```

Comment:
I think this should be `ogrsScore` to be consistent else it will lead to confusion

Reply:
Done